frontends/web/src/components/bottom-navigation/bottom-navigation.tsx

```diff
@@ -34,10 +39,14 @@ export const BottomNavigation = () => {
       </Link>
       <Link
         className={`${styles.link} ${pathname.startsWith('/account/') || pathname.startsWith('/accounts/') ? styles.active : ''}`}
-        to="/accounts/all"
+        to={
+          onlyHasOneAccount && accountCode ?
+            `/account/${accountCode}` :
+            '/accounts/all'
+        }
       >
         <AccountIconSVG />
-        {t('settings.accounts')}
+        {onlyHasOneAccount ? t('walletConnect.signingRequest.account') : t('settings.accounts')}
```

Comment:
Hm I am slightly not a fan of reuseing walletConnect translation here in bottom nav. Also we do reuse if from settings.accounts :) 

Could we add generic string somewhere in account. for account.account and account.accounts and then also reuse in settings and WC?